spec/features/user_behaviours_spec.rb

```diff
@@ -114,7 +114,7 @@
 private
   def sign_up_with(name, email, opts = nil)
     visit root_path
-    fill_in 'user_name', with: name
+    fill_in 'user_nickname', with: name
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.